core/components/molecules/_stack-layout/stack-layout.js

```diff
@@ -82,7 +82,7 @@ StackLayout.propTypes = {
   /** Horizontally distribute the components within the stack. applyes to `justify-content` */
   distribution: PropTypes.oneOf(['spaceBetween', 'center', 'start', 'end']),
   /** Flex wrap */
-  wrap: PropTypes.bool,
+  wrap: PropTypes.oneOf(['true', 'false']),
```

Comment:
@francocorreasosa Question: Why this change?

From a api usage point of view it'd feel more intuitive to use a `true,false` as booleans than `"true", "false"` as strings...

example

```jsx
<Stack wrap={true} >..</Stack>
```

vs

```jsx
<Stack wrap="false">..</Stack>
```

Reply:
Yeah, you're right. This is currently mapping 1:1 to HTML but we can make the change inside the component for sure.